processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -180,4 +223,27 @@ public boolean isClosed()
   {
     return !started.get();
   }
+
+  private List<LookupBean> getAllAsList()
```

Comment:
Thanks for th suggestion, i think ser deser of List is more intuitive and straightforward to jackson, i will prefer to keep it as list since a map won't provide an extra plus in my opinion.